topics/assembly/tutorials/chloroplast-assembly/tutorial.md

```diff
@@ -0,0 +1,312 @@
+> * Upload the `fasta` file there. Information about how to use the tool is available on the page.
+> * Once the annotation is completed, download the required files.
+> * In Galaxy, import the annotation `GFF3` file.
+{: .hands_on}
+
+Make a JBrowse file to view the annotations (the GFF3 file) under the assembly (the polished.fasta file).
+
+> ### {% icon hands_on %} Hands-on: View annotations
+> 1. **JBrowse genome browser** {% icon tool %}:
+>        - *"Reference genome to display"*: `Use a genome from history`
+>    - *"Select a reference genome"*: `polished-assembly.fasta`
+>    - *"Produce Standalone Instance"*: `Yes`
+>    - *"Genetic Code"*: `11. The Bacterial, Archaeal and Plant Plastid Code`
+>    - *"JBrowse-in-Galaxy Action"*: `New JBrowse instance`
+>    - *"Insert Track Group"*
+>        - *"Insert Annotation Track"*
+>            - *"Track Type"*: `GFF/GFF3/BED/GBK Features`
+>            - *"GFF/GFF3/BED Track Data"*: the `GFF3` file
+>            - *Leave the other track features as default*
+{: .hands_on}
+
+This may take a few minutes. There is one output file: re-name it `view-annotations`
+* Click on the eye icon to view.
+* Select the right contig to view, in the drop down box.
+* Zoom out (with the minus button) until annotations are visible.
+
+Your annotations may look like this:
+
+![annotations](../../images/annotations1.png "Annotations - zoomed out view")
+
+Zoom in (with the plus button) to see annotation details. Click on an annotation to see its sequence and source (e.g. the tool that predicted it).
+
+![annotations](../../images/annotations2.png "Annotations - zoomed in view")
+
+> ### {% icon question %} Question
+> Why might there be several annotations over the same genome region?
+> > ### {% icon solution %} Solution
+> > One reason is that these are predictions from different tools - such as BLAT or HMMER.
+> {: .solution}
+{: .question}
+
+# Repeat with new data
+
+*Optional extension exercise*
+
+We can assemble another chloroplast genome using sequence data from a different plant species: the snow gum, *Eucalyptus pauciflora*. This data is from {% cite Wang2018 %}. It is a subset of the original FASTQ read files (Illumina - SRR7153063, Nanopore - SRR7153095).
+
+> ### {% icon hands_on %} Hands-on: Assembly and annotation
+> * Get data: at this [Zenodo link](https://doi.org/10.5281/zenodo.3600662), then upload to Galaxy.
+> * Check reads: Run Nanoplot on the nanopore reads.
+> * Assemble: Use Flye to assemble the nanopore reads, then get Fasta statistics *Note: this may take several hours.*
+> * Polish assembly: Use Pilon to polish the assembly with short Illumina reads. *Note: Don't forget to map these Illumina reads to the assembly first using bwa-mem, then use the resulting `bam` file as input to Pilon.*
+> * Annotate: Use the GeSeq tool at [Chlorobox](https://chlorobox.mpimp-golm.mpg.de/geseq.html).
+> * View annotations:Use <ss>JBrowse to view the assembled, annotated genome.
```

Comment:
In order to repeat the analysis with different data, it states that each step should be repeated manually, instead of using the workflow previously generated.

Reply:
I'd rather keep it as it is to encourage repetition of steps, to cement the knowledge and skills.